package-lock.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "copy",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
please do not update the version

Reply:
@Prabhakar-Poudel if you check the diff, you can see that package.json is not at all touched, its the package-lock.json autogenerated by npm, and to be clear the version right now is the right version from your package.json.

For some reason the package-lock.json wasnt updated the last time you commited and hence the version is now udpated, If you check the package.json you can see the version is 1.1.0 (commited by you)

Reply:
Indeed. I see the version was not committed.

But I would slightly insist, that this is not the scope of the PR and the version update probably deserves another PR of its own to keep things separate.